development/dev-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nutsfoundation/nuts-node:master as nutsnode
+FROM debian:buster-slim
+
+RUN apt-get update && apt-get install -y curl sudo musl
+RUN curl -sL https://aka.ms/DevTunnelCliInstall | bash
+
+COPY entrypoint.sh /
+RUN chmod +x /entrypoint.sh
+
+COPY --from=nutsnode /usr/bin/nuts /usr/bin/nuts
+RUN chmod +x /usr/bin/nuts
```

Comment:
The number of layers needed can be reduced by moving stuff around and reducing the number of RUN commands.
Line 10-11 are the only parts that ever change, can they go at the very end?

Reply:
I don't think those last 2 layers (config) have a lot of influence on image building performance when cached.